tests/drivers/pwm/pwm_api/testcase.yaml

```diff
@@ -8,3 +8,16 @@ tests:
       or dt_alias_exists("pwm-3") or dt_compat_enabled("st,stm32-pwm")
       or dt_compat_enabled("intel,blinky-pwm") or dt_compat_enabled("nordic,nrf-pwm")
     depends_on: pwm
+  drivers.pwm.ke1xz_pwm_flexio:
+    tags:
+      - drivers
+      - pwm
+      - userspace
+    extra_args: DTC_OVERLAY_FILE="boards/frdm_ke1xz_flexio_pwm.overlay"
+    platform_allow:
+      - frdm_ke17z
+      - frdm_ke17z512
+    filter: (dt_alias_exists("pwm-0") or dt_alias_exists("pwm-1") or dt_alias_exists("pwm-2")
```

Comment:
why it requires pwm-0 or pwm-1 or pwm-2? do we test all those?

Reply:
These are required for acquiring pwm device nodes in test_pwm.c.
It is sufficient to test one, and they provide the same configuration.

![image](https://github.com/user-attachments/assets/5a957063-a29e-42b1-8fcd-715a00f64910)